docs/readmes/lte/upgrade_1_5.md

```diff
@@ -0,0 +1,28 @@
+---
+id: upgrade_1_5
+title: Upgrade to v1.5
+hide_title: true
+---
+
+# Upgrade to v1.5
+
+### Repo Change 
+
+In Magma Fuji (v1.5), Magma artifacts are now hosted on the new Magmacore repositories. 
```

Comment:
Can you add a direct link to the Magmacore repositories (homepage?). I think that will help operators feel more comfortable with it -- because right now, we're just asking them to blindly run a random script

On that note, can you also add a short snippet here on what/how the script actually does that